shared/copyq-completion

```diff
@@ -1,29 +1,29 @@
 # shellcheck shell=bash
 
-commands=(
+_copyq_commands=(
     show hide toggle menu exit disable enable clipboard selection paste copy
     count select next previous add insert remove edit separator read write
     action popup tab removetab renametab exporttab importtab config eval session
     help
 )
 
-list_sessions() {
-    ss -l -x | sed -nE 's#.*/\.copyq-?(\S*)_s .*#\1#p'
+_copyq_list_sessions() {
+    ss -l -x | sed -n '/\/\.copyq[^_]*_s /{s/.*\/\.copyq-\?\([^_]*\)_s .*/\1/;p}'
```

Comment:
Looks like this has been changed back accidentally.

Reply:
Woops. Fixed.